src/router/routes.js

```diff
@@ -353,10 +353,7 @@ module.exports = [
 	},
 	{
 		path: '/lending-home',
-		component: () => import('@/pages/LoanFinding/LoanFinding'),
-		meta: {
-			excludeFromStaticSitemap: true,
```

Comment:
Do we still need this entry for the old URL?

Reply:
Good question. I'm not sure b/c `/lend-by-category` route was not including that, if we want to preserve the same is not required I think.

```
meta: {
      unbounceEmailCapture: true,
},
```

But other cases like category page, /lend-by-category/*, includes both

```
excludeFromStaticSitemap: true,
unbounceEmailCapture: true,
unbouncePopUp: true,
```

What do you think about this?

Reply:
Right, I just think we might want to exclude the old URL in the static sitemap. Please take a look at where `excludeFromStaticSitemap` is used, since that might help determining the correct entry here. My assumption without testing is that `excludeFromStaticSitemap` will exclude `/lending-home` from the static site map. Or maybe the static sitemap excludes all redirects? That's generally what needs to be confirmed.

Reply:
We want `/lend-by-category` in the static sitemap but not `/lending-home`.

Reply:
If `/lending-home` is now a redirect you could leave in the excludeFromStaticSitemap setting but leave off the others as they signify something loaded in the client and that page would never be allowed to load due to the redirect.

Reply:
It looks like the other redirects don't have `excludeFromStaticSitemap`, but please confirm that the redirects are indeed excluded.

Reply:
Thanks @mcstover! Sounds like this redirect should still have `excludeFromStaticSitemap`.

Reply:
@mcstover Is there a reason why the other redirects don't have `excludeFromStaticSitemap`?

Reply:
I'm not sure. I think your point about running the script and verifying (or closely inspecting the sitemap generator code , NOT the UISiteMap.vue file) is the best way to know if it's required or not.

Reply:
I just took a look, and the sitemap lives locally at `http://localhost:8888/sitemaps/ui.xml`, and I verified in the that we do indeed ignore `redirect` entries in the generator that pulls from the Vue Router:

```
// Remove excluded routes
.filter(route => !route.meta?.excludeFromStaticSitemap && !route.redirect)
```

Reply:
@roger-in-kiva `excludeFromStaticSitemap` can be removed from this entry.